expected_results.yaml

```diff
@@ -54,11 +54,15 @@ testCases:
     - observability-crd-status
     - observability-pod-disruption-budget
     - observability-termination-policy
+    - operator-automount-tokens
     - operator-crd-openapi-schema
     - operator-crd-versioning
     - operator-install-source
     - operator-install-status-no-privileges
     - operator-install-status-succeeded
+    - operator-read-only-file-system
```

Comment:
I think this one will have to be moved to the `fail` section but let's see what the smoke tests say.